### PR TITLE
Bong Bread Smite Fix

### PR DESCRIPTION
**Pull request by tile4**
## About The Pull Request

Fixes an oversight of the initial PR

## Why It's Good For The Game

I would rather not leave the PR Idea half finished so I am going to try to complete it.

## Changelog
:cl:
tweak: tweaked a few things
admin: messed with admin stuff
/:cl:

### CODE REVIEW

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -1090,8 +1090,12 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	smite.effect(src, target)
 
 /proc/breadify(atom/movable/target)
-	var/obj/item/food/bread/plain/bread = new(get_turf(target))
-	target.forceMove(bread)
+	if(target/name == "Bong Bong")
```
Comment (Gboster-0) [inline code/modules/admin/verbs/randomverbs.dm:1093]:
```suggestion
	if(target.name == "Bong Bong")
```

File: code/modules/admin/verbs/randomverbs.dm [hunk 2]
```diff
@@ -1090,8 +1090,12 @@ Traitors and the like can also be revived with the previous role mostly intact.
...
+		var/obj/item/food/bread/plain/bread/bongbread = new(get_turf(target))
```
Comment (Gboster-0) [inline code/modules/admin/verbs/randomverbs.dm:1094]:
```suggestion
		var/obj/item/food/bread/plain/bongbread = new(get_turf(target))
```

Comment (Gboster-0) [inline code/modules/admin/verbs/randomverbs.dm:1094]:
```suggestion
		var/obj/item/food/bread/bongbread/bongbread = new(get_turf(target))
```

**Comment by tile4** [comment]
Honestly, I forgot about this PR. Fixed the things mentioned so hopefully it should be good?

**Review by Kitsunemitsu — APPROVED** [review]
Easy fix

**Pull request merged**